src/main/webapp/registration.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Registration Page</title>
+    <link rel="stylesheet" href="registration.css">
+    <script src="registration.js"></script>
```

Comment:
Kashish told me that if you put the script tag in the end of the file it will load faster. That would mean just adding this tag right before the end of the body tag. @kashiish

Reply:
Alright, fixed that